config/components.yml

```diff
@@ -26,9 +26,14 @@ containers:
   - "JavaBuildpack::Container::Tomcat"
 
 # In order to use Oracle JREs instead of OpenJDK, you must comment out the OpenJDK line and uncomment the Oracle line.
+# In order to use Zulu JREs instead of OpenJDK, you must comment out the OpenJDK line and uncomment the Zulu line.
+# To use Zulu JRE instead of OpenJDK without forking java-buildpack, set environment variable:
+# cf set-env <app_name> JBP_CONFIG_COMPONENTS '{jres: ["JavaBuildpack::Jre::ZuluJRE"]}'
+# cf restage <app_name>
 # Please see the documentation for more detail.
 jres:
   - "JavaBuildpack::Jre::OpenJdkJRE"
+# -  "JavaBuildpack::Jre::ZuluJRE"
```

Comment:
@Ekaterina001 These should be alphabetical (will also match documentation order).